mysql-test/suite/rocksdb/r/type_blob_indexes.result

```diff
@@ -24,15 +24,19 @@ INSERT INTO t1 (b,t,m,l) VALUES
 (REPEAT('b',128),REPEAT('f',128),REPEAT('e',128),REPEAT('d',128)),
 (REPEAT('c',128),REPEAT('b',128),REPEAT('c',128),REPEAT('e',128));
 EXPLAIN SELECT SUBSTRING(b,16) AS f FROM t1 WHERE b IN ('test1','test2') ORDER BY f;
-id	select_type	table	type	possible_keys	key	key_len	ref	rows	Extra
-#	#	#	#	#	PRIMARY	#	#	#	#
+id	select_type	table	partitions	type	possible_keys	key	key_len	ref	rows	filtered	Extra
+#	#	#	#	#	PRIMARY	#	#	#	#	#	Using where; Using filesort
```

Comment:
Ohh crap, good catch. I should have seen that as I actually did fix and re-record but must have not copied back to source tree or git add-ed.